content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -260,7 +260,7 @@ cd $DOWNLOAD_DIR
 sudo curl -L --remote-name-all https://storage.googleapis.com/kubernetes-release/release/${RELEASE}/bin/linux/${ARCH}/{kubeadm,kubelet,kubectl}
 sudo chmod +x {kubeadm,kubelet,kubectl}
 
-RELEASE_VERSION="v0.4.0"
+RELEASE_VERSION="$(curl -sSfL "https://api.github.com/repos/kubernetes/release/releases/latest"|grep tag_name|sed 's/^.*"tag_name": "\([^"]*\)".*$/\1/')"
```

Comment:
using a scriptable version fetch SGTM, but i think instead of `sed` and `grep` `jq` can be used here.

Reply:
It's fine to use `jq`; if switching, mention that some steps require the `jq` tool and link to it. It's not part of POSIX (yet :wink:)

Reply:
The extra mention should be at the top of the page in the prerequisites section.

Reply:
seems like `jq` is mentioned on a few other pages as well:
https://kubernetes.io/search/?q=jq

Reply:
It's fine to mention an option that uses `jq`, especially when we also provide a different option.

Reply:
I have updated the instructions to use `jq` instead